articles/aks/virtual-nodes-portal.md

```diff
@@ -33,7 +33,7 @@ Namespace                    RegistrationState
 Microsoft.ContainerInstance  Registered
 ```
 
-If the provider shows as *NotRegistered*, register the provider using the [az provider register][az-provider-register] as shown in the following example:
+If the provider shows as *NotRegistered*, register the provider using the `az provider register` as shown in the following example:
```

Comment:
Not sure where this was suppose to link to, but since it didn't I styled it using the code formatting